lms/djangoapps/instructor/views/api_urls.py

```diff
@@ -1,174 +1,174 @@
+        api.modify_access, name='modify_access'),
     url(r'^bulk_beta_modify_access$',
-        'lms.djangoapps.instructor.views.api.bulk_beta_modify_access', name="bulk_beta_modify_access"),
+        api.bulk_beta_modify_access, name='bulk_beta_modify_access'),
     url(r'^get_problem_responses$',
-        'lms.djangoapps.instructor.views.api.get_problem_responses', name="get_problem_responses"),
+        api.get_problem_responses, name='get_problem_responses'),
     url(r'^get_grading_config$',
-        'lms.djangoapps.instructor.views.api.get_grading_config', name="get_grading_config"),
+        api.get_grading_config, name='get_grading_config'),
     url(r'^get_students_features(?P<csv>/csv)?$',
-        'lms.djangoapps.instructor.views.api.get_students_features', name="get_students_features"),
+        api.get_students_features, name='get_students_features'),
     url(r'^get_issued_certificates/$',
-        'lms.djangoapps.instructor.views.api.get_issued_certificates', name="get_issued_certificates"),
+        api.get_issued_certificates, name='get_issued_certificates'),
     url(r'^get_students_who_may_enroll$',
-        'lms.djangoapps.instructor.views.api.get_students_who_may_enroll', name="get_students_who_may_enroll"),
+        api.get_students_who_may_enroll, name='get_students_who_may_enroll'),
     url(r'^get_user_invoice_preference$',
-        'lms.djangoapps.instructor.views.api.get_user_invoice_preference', name="get_user_invoice_preference"),
+        api.get_user_invoice_preference, name='get_user_invoice_preference'),
     url(r'^get_sale_records(?P<csv>/csv)?$',
-        'lms.djangoapps.instructor.views.api.get_sale_records', name="get_sale_records"),
+        api.get_sale_records, name='get_sale_records'),
     url(r'^get_sale_order_records$',
-        'lms.djangoapps.instructor.views.api.get_sale_order_records', name="get_sale_order_records"),
+        api.get_sale_order_records, name='get_sale_order_records'),
     url(r'^sale_validation_url$',
-        'lms.djangoapps.instructor.views.api.sale_validation', name="sale_validation"),
+        api.sale_validation, name='sale_validation'),
     url(r'^get_anon_ids$',
-        'lms.djangoapps.instructor.views.api.get_anon_ids', name="get_anon_ids"),
+        api.get_anon_ids, name='get_anon_ids'),
     url(r'^get_student_progress_url$',
-        'lms.djangoapps.instructor.views.api.get_student_progress_url', name="get_student_progress_url"),
+        api.get_student_progress_url, name='get_student_progress_url'),
     url(r'^reset_student_attempts$',
-        'lms.djangoapps.instructor.views.api.reset_student_attempts', name="reset_student_attempts"),
+        api.reset_student_attempts, name='reset_student_attempts'),
     url(
         r'^rescore_problem$',
-        'lms.djangoapps.instructor.views.api.rescore_problem',
-        name="rescore_problem"
+        api.rescore_problem,
+        name='rescore_problem'
```

Comment:
nit: a fair number of these now seem short enough to easily fit on one line, not sure if it's worth the effort of condensing them

Reply:
I didn't want to complicate an already very busy diff, but since you've reviewed it I'll do that now.